enum/CloseMessages.ts

```diff
@@ -0,0 +1,7 @@
+export enum CloseMessages {
+	CLOSED_BY_AGENT = 'Closed by agent',
+	CLOSED_BY_VISITOR = 'Closed by visitor',
+	CLOSED_DUE_TO_TIMEOUT = 'Chat closed due to timeout',
```

Comment:
For consistency, can we change `Closed by agent` and `Closed by visitor` to `Chat closed by agent` `Chat closed by visitor`